Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -342,32 +369,42 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         let vendorOptions = provider?.providerConfiguration
 
         loadRoutes(from: vendorOptions)
-        self.isConnectionTesterEnabled = vendorOptions?[NetworkProtectionOptionKey.connectionTesterEnabled] as? Bool ?? true
```

Comment:
This should not have been in vendor options.  In fact it was not disabled in iOS due to this.

I've moved it to line 362 so it's part of the startup options instead.